### ISSUE TITLE
Channeling Bar not showing

### ISSUE THREAD

**Issue by chrisgle** [Aviana/LunaUnitFrames#326]
When I am fishing, not channeling bar appears. My castbar option is checked.

**Comment by Aviana** [comment]
Bar is showing correctly however it was using the cast color not the channeling one.
![fishing](https://user-images.githubusercontent.com/425023/32509411-7f72451a-c3ed-11e7-97b4-d103431d320b.png)

**Comment by chrisgle** [comment]
Hi again.

It still doesn't work in my configuration. Using latest version (v2224). Erased all luna config files. Disabled all addons. Still doesn't work. When I use version 2210, it appears (I simply rename the folder to change between versions).

Any ideas?

**Comment by raethkcj** [comment]
On the latest version (v2225) on Kronos 3 I am not seeing a castbar for any channeled abilities.

I was able to get it to work for myself by adding the following bolded line in cast.lua, although I don't know enough about Luna to be sure that won't break something else.
<pre>
666   else
667     <b>if (event == "SPELLCAST_STOP" and frame.castBar.channeling) then return end</b>
668     if (event == "SPELLCAST_CHANNEL_STOP" and not frame.castBar.channeling) then return end
669     if (event == "SPELLCAST_FAILED" and frame.castBar.channeling) then return end
670     frame.castBar.casting = false
671     frame.castBar.channeling = false
672     frame.castBar:SetScript("OnUpdate", nil)
673     Cast:FullUpdate(frame)
674   end
</pre>

When casting a channeled ability, I noticed I was getting two events, `SPELLCAST_CHANNEL_START` and a `SPELLCAST_STOP`. The `SPELLCAST_STOP` was setting channeling to false.
